src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -38,9 +38,12 @@ def to_internal_value(self, data):
 
 class SchemaField(serializers.Field):
     def to_internal_value(self, data):
-        if not data:
+        if data is None:
             return
 
+        if data == '' or data == {}:
+            return {}
```

Comment:
See: above question. If not, we could collapse all of this into:

```python
if not data:
    return {}
```

since `{}` and `''` are `None`.